scripts/staging/mirror-images.sh

```diff
@@ -6,11 +6,12 @@
 set -euo pipefail
 
 SCRIPT_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]:-"$0"}")")"
+CURRENT_ROOT_DIR="${SCRIPT_DIR}/../.."
```

Comment:
If using the term "current", then it should be the other way around?

Reply:
Could be, should I do that then?

Reply:
PARENT_ROOT_DIR seems fine